bot/code_review_bot/__init__.py

```diff
@@ -26,6 +26,13 @@
 COVERITY = "coverity"
 
 
+def positive_int(x):
+    """Helper to get a positive integer or None"""
```

Comment:
I'd warn when the line is negative, so we can fix the problem in the analyzers (and make them return None for full file issues).

Reply:
You smart :cookie: